samples/highcharts/blog/synchronize-axes-multiple-charts/demo.html

```diff
@@ -0,0 +1,56 @@
+<script src="https://code.highcharts.com/highcharts.js"></script>
+<script src="https://code.highcharts.com/modules/data.js"></script>
+<script src="https://code.highcharts.com/modules/exporting.js"></script>
+<script src="https://code.highcharts.com/modules/accessibility.js"></script>
+
+<div class="chart" id="container1"></div>
+<div class="flex-container">
+  <div class="chart" id="container2"></div>
+  <div class="chart" id="container3"></div>
+</div>
+<div class="chart" id="container4"></div>
+
+<!-- Highcharts plugin that makes all chart's x and y axes to follow the main chart's axes. -->
+<script>
```

Comment:
Why is this script in the demo.html instead of demo.js? If it is to make a clear separation between the plugin and the setup code I think we should do it with comments instead, like we currently do with other similar cases.